ci/request-reviews/default.nix

```diff
@@ -0,0 +1,43 @@
+{
+  lib,
+  stdenvNoCC,
+  makeWrapper,
+  coreutils,
+  codeowners,
+  jq,
+  curl,
+  github-cli,
+  gitMinimal,
+}:
+stdenvNoCC.mkDerivation {
+  name = "request-reviews";
+  src = lib.fileset.toSource {
+    root = ./.;
+    fileset = lib.fileset.unions [
+      ./get-reviewers.sh
+      ./request-reviews.sh
+      ./verify-base-branch.sh
+      ./devBranches.txt
```

Comment:
Nit: If the other files in this directory use [`kebab-case`](https://en.wikipedia.org/wiki/Letter_case#Kebab_case), this one shouldn't use [`camelCase`](https://en.wikipedia.org/wiki/Camel_case).